src/clevertap.js

```diff
@@ -26,6 +26,7 @@ export default class CleverTap {
     }
     Account.setAccountId(id);
     Account.setRegion(region);
+    Device.setVAPIDState(false);
```

Comment:
Let's not set this to false on initialisation. If the vapid was fetched in previous app launch then we can continue with registering without waiting for the same token to be fetched.

Reply:
@royson-ct Got it, I'll check for the state key in StorageManager before setting this as true/false